spec/string_spec.rb

```diff
@@ -29,4 +31,35 @@
     end
   end
 
+  it "replaces spaces with underscores" do
+    Benchmark.ips do |x|
+      str = Random.new.bytes(1024)
```

Comment:
Will a random string like this be sufficient? I wasn't sure if I should be testing with a string that is guaranteed to have the characters we are testing replacement on?